### PR TITLE
James 2292 patch mailqueue

### PR DESCRIPTION
**Pull request by mbaechler**

### CODE REVIEW

File: server/protocols/webadmin/webadmin-mailqueue/src/main/java/org/apache/james/webadmin/routes/MailQueueRoutes.java [hunk 1]
```diff
@@ -241,4 +249,78 @@ private boolean filter(MailQueueItemDTO item, Optional<Boolean> isDelayed) {
             .map(delayed -> delayed == mailIsDelayed)
             .orElse(true);
     }
+
+    @PATCH
+    @Path("/{mailQueueName}/mails")
+    @ApiImplicitParams({
+        @ApiImplicitParam(required = true, dataType = "string", name = "mailQueueName", paramType = "path"),
+        @ApiImplicitParam(required = false, dataType = "boolean", name = DELAYED_QUERY_PARAM, paramType = "query")
```
Comment (aduprat) [inline server/protocols/webadmin/webadmin-mailqueue/src/main/java/org/apache/james/webadmin/routes/MailQueueRoutes.java]: You can reuse the documentation from the GET endpoint
```
        @ApiImplicitParam(
                required = false, 
                dataType = "boolean", 
                name = DELAYED_QUERY_PARAM, 
                paramType = "query",
                example = "?delayed=true",
                value = "Whether the mails are delayed in the mail queue or not (already sent).")
```

File: server/queue/queue-memory/src/main/java/org/apache/james/queue/memory/MemoryMailQueueFactory.java [hunk 2]
```diff
@@ -255,9 +254,10 @@ public void done(boolean success) throws MailQueue.MailQueueException {
             }
         }
 
+
```
Comment (rouazana) [inline server/queue/queue-memory/src/main/java/org/apache/james/queue/memory/MemoryMailQueueFactory.java]: superfluous line?

**Comment by chibenwa** [comment]
Unused imports on the ci in jms project. Can someone check?

**Comment by rouazana** [comment]
push -f to squash and to rename the first commit (incorrect jira number)

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**